tests/integration/server/courseInstance/courseInstance.controller.test.ts

```diff
@@ -1046,18 +1048,20 @@ describe('CourseInstance API', function () {
     let newOfferedValue: OFFERED;
     let response: Response;
     let testCourseInstance: CourseInstance;
+    const currentAcademicYear = 2022;
```

Comment:
This not that important but I prefer if you move `const currentAcademicYear = 2022;` to the top behind the `let configService: ConfigService;`. Still if you like to leave it like this then its fine.

Reply:
Sure, `currentAcademicYear` was moved to be after `configService.`